src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -648,17 +650,35 @@ int goto_instrument_parse_optionst::doit()
     {
       do_indirect_call_and_rtti_removal();
       call_grapht call_graph(goto_model);
-
       if(cmdline.isset("xml"))
         call_graph.output_xml(std::cout);
-      else if(cmdline.isset("dot"))
-        call_graph.output_dot(std::cout);
       else
-        call_graph.output(std::cout);
+        call_graph.output_dot(std::cout);
 
```

Comment:
In some ways it would be nice to keep output as text, output_dot as dot (and requiring the dot option), output_xml as xml (and requiring the xml option) and so on.